src/components/Editor/Breakpoint.js

```diff
@@ -41,7 +42,7 @@ const Breakpoint = React.createClass({
     }
   },
 
-  shouldComponentUpdate(nextProps) {
+  shouldComponentUpdate(nextProps: any) {
```

Comment:
Wouldn't `nextProps` here always be an object with the breakpoints and editor since they're required props?